cmd/ttn-lw-cli/commands/end_device_templates.go

```
@@ -295,6 +307,7 @@ This command takes end device templates from stdin.`,
 		Short:             "Convert data to an end device template (EXPERIMENTAL)",
 		PersistentPreRunE: preRun(),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			optionalAuth()
```

Use `_ = func()` to discard the error explicitly.